components/ActionSheet/README.md

```diff
@@ -37,8 +37,10 @@ the screen and displays actions a user can take.
   - [Typical use](#typical-use)
 - [MDCActionSheetController vs. UIAlertControllerStyleActionSheet](#mdcactionsheetcontroller-vs.-uialertcontrollerstyleactionsheet)
 - [Extensions](#extensions)
-  - [Color Theming](#color-theming)
-  - [Typography Theming](#typography-theming)
+  - [Theming](#theming)
+  - [How to theme an MDCActionSheet](#how-to-theme-an-mdcactionsheet)
+  - [Color Theming (To be deleted)](#color-theming-(to-be-deleted))
+  - [Typography Theming (To be deleted)](#typography-theming-(to-be-deleted))
```

Comment:
Looks like this link needs to be fixed as well. (https://github.com/material-components/material-components-ios/issues/7068)